src/components/Blocks/Introduction/Edit.jsx

```diff
@@ -2,13 +2,25 @@ import React from 'react';
 import { SidebarPortal } from '@plone/volto/components';
 import Sidebar from './Sidebar';
 import { DetachedTextBlockEditor } from '@plone/volto-slate/blocks/Text/DetachedTextBlockEditor';
+import config from '@plone/volto/registry';
 
 const IntroductionBlockEdit = (props) => {
   const { data, onChangeBlock, block, selected } = props;
+  const unwantedButtons = ['heading-two', 'heading-three', 'blockquote'];
+
+  const filteredToolbarProps = {
+    ...props,
+    slateSettings: {
+      ...config.settings.slate,
+      toolbarButtons: config.settings.slate.toolbarButtons.filter(
+        (button) => !(unwantedButtons || []).includes(button),
```

Comment:
This can just be

```suggestion
        (button) => !unwantedButtons.includes(button),
```

since unwantedButtons is always set to the same literal value above